students/steve-long/lesson01-enviro/activities-exercises/task-1-exploring-errors/break_me.py

```diff
@@ -0,0 +1,89 @@
+# Python210 | Fall 2020
+# ----------------------------------------------------------------------------------------
+# Lesson01
+# Task 1: Exploring Errors (break_me.py)
+# Steve Long 2020-09-10
+
+# /Users/steve/Documents/Project/python/uw_class/python210/lessons/lesson01-enviro/break_me.py
+
+def runWithNameError():
```

Comment:
All function names should be snake_case in Python: run_with_name_error():

Reply:
Looks like indentations are over indented too.  Make sure your text editor is set up for 4 spaces.